README.md

```diff
@@ -172,11 +172,18 @@ There are 5 different environment tiles, of which some have different styles of
 #### Locations
 As of January 2022, there are a total of 29 different locations in game.
 
-A lot of the earlier details such as what order which locations were unlocked or released in are no longer easily found without restarting the game, however looking at the game's [version history][history], the rough suggested order is:
+A lot of the earlier details such as what order which locations were unlocked or released in are no longer easily found without restarting the game, however looking at the game's [version history][history], the suggested order is:
```

Comment:
Thanks for correcting this! I think this would make more grammatical sense now.
```suggestion
A lot of the earlier details such as what order which locations were unlocked or released in are no longer easily found without restarting the game, however looking at the game's [version history][history], the determined is:
```